apollo-router-core/src/request.rs

```diff
@@ -46,6 +46,34 @@ where
 }
 
 impl Request {
+    pub fn from_urlencoded_query(url_encoded_query: String) -> Result<Request, serde_json::Error> {
+        // decode percent encoded string
+        // from the docs `Unencoded `+` is preserved literally, and _not_ changed to a space.`,
```

Comment:
Which docs? Specify a reference here please.

Reply:
Also probably could do with a larger comment here with background on the problem.